libvmi/driver/kvm/libvirt_wrapper.c

```diff
@@ -68,5 +82,13 @@ status_t create_libvirt_wrapper(kvm_instance_t *kvm)
     wrapper->virDomainResume = dlsym(wrapper->handle, "virDomainResume");
     wrapper->virConnectAuthPtrDefault = dlsym(wrapper->handle, "virConnectAuthPtrDefault");
 
-    return sanity_check(kvm);
+    wrapper->virDomainQemuMonitorCommand = dlsym(wrapper->handle_qemu, "virDomainQemuMonitorCommand");
+
+    int ret = sanity_check(kvm);
```

Comment:
sanity_check returns a `status_t`, GCC doesn't give you any warnings here ?

Reply:
took remark into account

Reply:
Good point, but there was no warning here. I corrected type.